src/N3Store.js

```diff
@@ -775,6 +771,158 @@ export default class N3Store {
     return lists;
   }
 
+  /**
+   * Returns `true` if the current instance is a superset of the given dataset; differently put: if the given dataset
+   * is a subset of, is contained in the current dataset.
+   *
+   * Blank Nodes will be normalized.
+   */
+  addAll(quads) {
+    if (Array.isArray(quads))
+      this.addQuads(quads);
+    else {
+      for (const quad of quads)
+        this.add(quad);
+    }
+    return this;
+  }
+
+
+  /**
+   * Returns `true` if the current instance is a superset of the given dataset; differently put: if the given dataset
+   * is a subset of, is contained in the current dataset.
```

Comment:
```suggestion
   * Returns `true` if the current dataset is a superset of the given dataset; in other words, returns `true` if
   * the given dataset is a subset of, i.e., is contained within, the current dataset.
```

Reply:
@TallTed all phrasing for JSDoc in this PR is taken directly from https://rdf.js.org/dataset-spec/#dataset-interface. You may wish to reflect these changes over there.